src/components/assets/Erc20Currency.vue

```diff
@@ -35,11 +35,15 @@
             >
               {{ $t('assets.transfer') }}
             </button>
-            <div v-if="token.isWrappedToken">
+            <div v-if="token.isWrappedToken && !token.isXC20">
```

Comment:
![Screen Shot 2022-05-16 at 8 58 40 AM](https://user-images.githubusercontent.com/40356749/168536543-ead1d815-b08e-47cf-9187-060887d286e9.png)

Since we separate the asset view in the native assets list, how about we create another section for XC20 assets too so the view is symmetric?

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/398/commits/d10010f5a6d9dbe199597a63ebd80ade3916ed5d) commit.

![image](https://user-images.githubusercontent.com/92044428/168569967-29a479b9-1949-4a09-81c7-731620ae9fd7.png)